curriculum/challenges/english/10-coding-interview-prep/rosetta-code/comma-quibbling.md

```diff
@@ -8,29 +8,18 @@ dashedName: comma-quibbling
 
 # --description--
 
-Comma quibbling is a task originally set by Eric Lippert in his [blog](https://blogs.msdn.com/b/ericlippert/archive/2009/04/15/comma-quibbling.aspx).
+[Comma quibbling](https://rosettacode.org/wiki/Comma_quibbling) is a task originally set by Eric Lippert in his 2009 blog. In this challenge, you will create a `string` from an `array`. You have to account for the `array` having no items, a single item, or multiple items in it. 
 
 # --instructions--
 
-Write a function to generate a string output which is the concatenation of input words from a list/sequence where:
+Write a function to generate a string output which is the concatenation of input words from a Array where:
 
-<ol>
-  <li>An input of no words produces the output string of just the two brace characters (<code>"{}"</code>)</li>
-  <li>An input of just one word, e.g. <code>["ABC"]</code>, produces the output string of the word inside the two braces, e.g. <code>"{ABC}"</code></li>
-  <li>An input of two words, e.g. <code>["ABC", "DEF"]</code>, produces the output string of the two words inside the two braces with the words separated by the string <code>" and "</code>, e.g. <code>"{ABC and DEF}"</code></li>
-  <li>An input of three or more words, e.g. <code>["ABC", "DEF", "G", "H"]</code>, produces the output string of all but the last word separated by <code>", "</code> with the last word separated by <code>" and "</code> and all within braces; e.g. <code>"{ABC, DEF, G and H}"</code></li>
-</ol>
-
-Test your function with the following series of inputs showing your output here on this page:
-
-<ul>
-  <li>[] # (No input words).</li>
-  <li>["ABC"]</li>
-  <li>["ABC", "DEF"]</li>
-  <li>["ABC", "DEF", "G", "H"]</li>
-</ul>
-
-**Note:** Assume words are non-empty strings of uppercase characters for this task.
+|             Array          |           Output         |
+| -------------------------- | ------------------------ |
+| []                         | "{}"                     |
+| ["ABC"]                    | "{ABC}"                  |
+| ["ABC", "DEF"]             | "{ABC and DEF}"          |
+| ["ABC", "DEF", "G", "H"]   | "{ABC, DEF, G and H}"    |
```

Comment:
```suggestion
| []                         | {}                       |
| ["ABC"]                    | {ABC}                    |
| ["ABC", "DEF"]             | {ABC and DEF}            |
| ["ABC", "DEF", "G", "H"]   | {ABC, DEF, G and H}      |
```

Pretty minor, but the quotes didn't seem like the belonged. What do you think @Sboonny?